src/db/index.ts

```diff
@@ -1,11 +1,13 @@
 import { MongoClient } from "mongodb";
 
 export default async function initialiseDb(url: string, databaseName: string, cert?: (string | Buffer)[]) {
+  console.log('Connecting to DB...')
   const client = await MongoClient.connect(url, { 
     sslValidate: !!cert,
     sslCA: cert,
     useNewUrlParser: true,
     useUnifiedTopology: true 
   });
+  console.log('Connected to DB.');
   return client.db(databaseName);
```

Comment:
Not to fix now, but something to address later. I think this should just return the client, as to close the connection you need to use the client.